docs/migrating-v1alpha1-to-v1beta1.md

```diff
@@ -60,11 +59,152 @@ spec:
     value: https://example.com/foo.json
 ```
 
-#### Input and Output PipelineResources
+## PipelineResources and Catalog Tasks
+
+`PipelineResources` are remaining in alpha while the other resource kinds are
+promoted to beta. This is part of a deliberate effort to migrate users away
+from `PipelineResources` onto a broader suite of reusable `Tasks` provided by
+the Tekton catalog.
+
+_More on the reasoning and what's left to do in
+[Why aren't PipelineResources in Beta?](docs/resources.md#why-arent-pipelineresources-in-beta)._
+
+In the near-term `PipelineResources` will continue to be supported by Tekton.
```

Comment:
Do we want to use `supported` here ?

Reply:
Until there's a definite path away / around them I kinda figure I'll be continuing to review and merge fixes for the existing types and will try to provide help for PipelineResource issues when I have the time for it.  But I would push back against anyone who wants to add _new_ PR types.  There was a suggestion of an SVN PipelineResource recently and I pushed back against that, for example.  `supported` seems right to me?

Reply:
I'm updating this to try to say more of what @sbwsg said instead of "supported"